scss/ui/partials/ui/onboarding/_ngonboarding.scss

```diff
@@ -185,9 +183,6 @@
 }
 
 .onboarding-focus {
-  // sass-lint:disable no-vendor-prefixes
-  -moz-transition: none !important;
-  -webkit-transition: none !important;
```

Comment:
Same for these. There isn't anything wrong with having vendor prefixes, we just disallow them in the linter because they should be provided via a single mixin line from Compass.

Reply:
I removed these because they're overridden by the following transition line which is supported for our required browsers.

Reply:
That's fine. I was thinking we should replace all vendor prefixes (and their non-prefixed counterparts) with Compass mixins then rely on [tuning vendor prefix support](http://compass-style.org/help/documentation/tuning-vendor-prefixes/) to control which prefixes make it into the compiled CSS.